### DISCUSSION TITLE
RPC error: details='exception calling application: list index out of range'

### DISCUSSION THREAD

**Discussion by Chisdo** [milvus-io/milvus#6019] [Q&A and General discussion]
Configures:
1. Docker-enabled Mishards: v1.1.0:
2. GPU-wr-1.1.0, CPU-ro-1.1.0 * 8
3. Collection info: {'index_file_size': 1024, 'index_type': IndexType.IVF_SQ8, 'dimension': 480}
4. status, dst_res = m.search(
            collection_name='collection20210622',
            partition_tags=['_defaults'],
            query_records=[[random() for _ in range(480)]],
            top_k=10000,
            params={'nprobe': 47}
        )
During the search conduct, I met this problem：RPC error: details='exception calling application: list index out of range'
Please let me know if you need more information about reproducing the problem and how can I get those infos.

<img width="1253" alt="Screen Shot 2021-06-23 at 16 07 44" src="https://user-images.githubusercontent.com/36720318/123060325-34459100-d43d-11eb-9c1b-3651b5a6ed82.png">

**Comment by yhmo** [comment]
The mishards version and milveus version is?

**Reply by Chisdo** [reply]
all of them are v1.1.0

**Comment by yhmo** [comment]
Seems it is this issue #5191, which is fixed in v1.1.1(mishards).

**Reply by Chisdo** [reply]
OK, I will look into it. Thx a lot!
